api.js

```diff
@@ -31,6 +31,11 @@ function Api(files, options) {
 	this.tests = [];
 	this.files = files || [];
 
+	if (this.require && !Array.isArray(this.require)) {
+		// normalise single `--require moduleId` usage
+		this.require = [this.require];
+	}
```

Comment:
Just use [`arrify`](https://github.com/sindresorhus/arrify) on line 69 and drop this whole thing.